pkgs/development/python-modules/drf-extra-fields/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, setuptools
+, django
+, djangorestframework
+, filetype
+}:
+
+buildPythonPackage rec {
+  pname = "drf-extra-fields";
+  version = "3.7.0";
+  pyproject = true;
+
+  src = fetchPypi {
```

Comment:
How about using `fetchFromGitHub`? Using it makes it easier to track changes in the source code.